build/dangerjs/detectChangedDependencies.ts

```diff
@@ -0,0 +1,98 @@
+import * as _ from 'lodash'
+import { DangerDSLType } from 'danger'
+
+type DangerJS = {
```

Comment:
Can we extract this type and use it everywhere, i.e. `checkChangelog()`, `detectChangedDependencies()`, etc?